newIDE/app/src/UI/Theme/DefaultTheme/index.js

```diff
@@ -71,6 +71,8 @@ const gdevelopTheme = {
     separatorColor: '#e0e0e0',
     selectedBackgroundColor: systemSelectionColor,
     selectedTextColor: '#ffffff',
+    rightIconColor: '#111',
+    selectedRightIconColor: '#111',
```

Comment:
I've added two more gdevelop specific variables to change the color of secondary action icons (for example the `more menu` icon of object list), we can get this result now:

![Screenshot from 2020-05-03 19-46-48](https://user-images.githubusercontent.com/39851078/80916531-e7a18100-8d76-11ea-9ea5-a76ce2a99e25.png)

Reply:
Mmmm ok I get it! Can you verify what you entered for the other themes though? I'm not sure about the color that were put there.

Reply:
> Can you verify what you entered for the other themes though?

There was nothing explicitly mentioned, but I can verify that they should be the correct colors, as I can see them being overridden here,
https://github.com/4ian/GDevelop/blob/d6f99c5841d584021993c78ea8ee5f6268583c2e/newIDE/app/src/UI/Theme/DefaultTheme/index.js#L136-L147

Reply:
Indeed so this is a color used in the default theme, but in your changes I see it being used for the Dark theme too - is that normal?

Reply:
Would you be able to check if it's not something doable with material-ui styles directly? https://material-ui.com/api/list-item/
Let me know. As much as possible, I prefer to avoid overriding the color/style of Material-ui components directly inside our components, because it's making us diverge a bit more from material-ui, which might expose already things in `createMuiTheme` (which remove a bit of code complexity compared to passing "yet another" prop).

Reply:
> but in your changes I see it being used for the Dark theme too - is that normal?

Oh okay I get it, dark theme doesn't override icon colors and checking on the gdevelop online editor I can see that it uses the usual white color, makes sense :sweat_smile: I'll fix this right now :)

![Screenshot from 2020-05-03 20-19-16](https://user-images.githubusercontent.com/39851078/80917263-6ac4d600-8d7b-11ea-9339-c20dd57e3b92.png)

Reply:
> Would you be able to check if it's not something doable with material-ui styles directly?

Maybe there is a way, we can override `selected` style of `ListItem`. This way, I guess we can get rid of the entire listItem gdevelop variables and handle everything with `createMuiTheme`? I'll give it a try and let you know :)

Reply:
> Maybe there is a way, we can override `selected` style of `ListItem`.

@4ian I can't find any way with this at all, the styles of `selected` are passed through the theme context but they are never applied on the element. I guess this is an issue with MUI itself. Also even if I manage to apply the stylings to the element, I still have to target a nested component (`selected` is a rule of `ListItem`, I have to change the style of `SvgIcon` which is nested withing `ListItem`.)

Also, I can't directly change the style of `SvgIcon` because it is dependent upon the state of `ListItem` :sweat:. 

Another possible solution I think is using a context on SvgIcon so that we don't have to pass any props, but I guess this would be overkill just to change colour.

Reply:
Ok, thanks for researching this. In this case don't worry and let's keep these custom styles :)